src/utils/validation.ts

```diff
@@ -61,6 +61,9 @@ export const stringOrNumberResolverFactory = <FormData, CustomContext = {}>(
 
 export function formatSchemaErrorMessage(errorString?: string): string | undefined {
   if (!errorString) return undefined
-  const cleanedString = errorString.replace(/"/g, '')
-  return cleanedString[0].toUpperCase() + cleanedString.slice(1)
+  const cleanedString = errorString
+    .split(' ')
+    .slice(1)
+    .join(' ')
```

Comment:
This looks strange. What `errorString` do you get for input and what do you expect as output?

Reply:
It is exactly: `"{{keyName}}" {{restOfErrorMessage}}`

e.g `"sellToken value must be positive"`

which is ugly

Reply:
then
```suggestion
  const cleanedString = errorString.replace(/^"\w+"\s+/, '')
```

Reply:
no like i dont want the key name at all

Reply:
```
`"sellToken" value must be positive`.replace(/^"\w+"\s+/, '') === "value must be positive"